tests/test_plotting.py

```diff
@@ -215,7 +215,7 @@ def test_qt_interactor(qtbot):
     vtk_widget = window.vtk_widget  # QtInteractor
 
     # check that QtInteractor.__init__() is called
-    assert_hasattr(vtk_widget, "iren", vtk.vtkRenderWindowInteractor)
+    hasattr(vtk_widget, "iren")
```

Comment:
I think this has become a noop, did you mean `assert hasattr`?

Reply:
Yes exactly!